### ISSUE TITLE
[Accessibility]: "Type to filter by filename" is announced twice in ref panel

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36605]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33510

### Problem description

When navigating through the ref panel, "Type to filter by filename" is announced twice, once near the close button and once on the actual text field.

### Expected behavior

"Type to filter by filename" should be announced only on the text field. If there is some need to announce the whole thing, it should say "reference and implementation panel" or something like that.

### Additional details

https://user-images.githubusercontent.com/93103176/171984181-bd7533b6-15c8-4377-a9bc-3a97fe0d164c.mov

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
